appcli/templates/installer.j2

```diff
@@ -85,8 +97,27 @@ function print_error_and_exit()
     exit 1
 }
 
+function args()
+{
+    options=$(getopt --long auto-configure --long no-auto-configure -- "$@")
```

Comment:
`getopt` is not a built in command, it's an external command which is not POSIX compliant.
It behaves differently on various Linux distros (e.g. on `Arch` the `--long` option is called `--longoptions`.
So this isn't very portable which is problematic as we need the script to be as portable as possible.
I'll update to reflect how we parse long args in our plummet example script.